app/src/main/res/drawable/ic_directions_black_24dp.xml

```diff
@@ -0,0 +1,13 @@
+<!--
```

Comment:
No need for size in vector title `ic_directions_black_24dp.xml` -> `ic_directions_black`. Might not even need colour?